neuralforecast/data/hiertsdataset.py

```diff
@@ -0,0 +1,119 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/data__hiertsdataset.ipynb (unless otherwise specified).
+
+__all__ = ['HierTimeseriesDataset']
+
+# Cell
+import numpy as np
+from torch.utils.data import Dataset, DataLoader
+
+# Cell
+class HierTimeseriesDataset(Dataset):
+    def __init__(self,
+                 # Bottom data
+                 Y, X, S,
```

Comment:
I think it would be better if the input names match the [`TimeSeriesDataset`](https://github.com/Nixtla/neuralforecast/blob/main/neuralforecast/data/tsdataset.py): `Y_df`, `X_df`, `S_df`.